packages/@o3r/core/schematics/rule-factories/generic-updates/index.ts

```diff
@@ -20,6 +23,29 @@ ${folderToExclude}
         `;
         }
       });
+      const packageManager = getPackageManager({workspaceConfig});
+      if (packageManager === 'yarn') {
+        gitignore += `
+
+### yarn ###
```

Comment:
Question (that is not directly relative this PR): Do we execute a `yarn set version @${version}` before generating a yarn2+ gitigore?`

Reply:
Indeed, we do not execute.
I have created an issue about (to fix the yarn version inside the repository for standalone sdks).
https://github.com/AmadeusITGroup/otter/issues/940